lib/log.js

```diff
@@ -192,11 +192,44 @@ Lug.prototype.activityEvent = function (event, request, data) {
       })
 
       self.logger.info('activityEvent', info)
+      // log the same event with a different name
+      // strip the uid, instead we have the flow_id
+      delete info.uid
+      self.logger.info('flowEvent', info)
       self.statsd.write(info)
     }
   )
 }
 
+// Log a flow metrics event.
+// These events help understand the user flow.
+
+Lug.prototype.flowEvent = function (event, request) {
```

Comment:
Just noting for the future that there is a lot of duplication between this method and `log.activityEvent` now. If we were to migrate all of the `flowId` / `metricsContext` stuff purely into the flow events, it would eliminate much of this duplication.

Alternatively, we could just refactor the duplication to a common helper.

Neither of these suggestions are for this PR, I just wanted a comment I can link to from elsewhere when we properly start working on the sign-in funnel.